sneak_peek.md

```diff
@@ -3,9 +3,9 @@ This is essentially a placeholder for the next release note ...
 ------------------------------------------------------------------------------
 
 * New features
-  + ADIOS driver -- Read ADIOS 1.x BP formated file. 
-    ADIOS_READ_METHOD_BP must be set when open BP file.
-    Does not support low-level and non blocking API.
+  + ADIOS driver -- Reading ADIOS 1.x BP formated file. 
```

Comment:
New feature is not the driver. Drivers are implementations to enable features.
New feature is the read capability of BP files.